content/ADR-244-extended-wearables-addressable.md

```diff
@@ -0,0 +1,59 @@
+
+To address this limitation and enhance the representation of wearable NFT ownership, we propose an extension to the content-addressable URN format specifically tailored for minted wearables. This extension introduces an optional token ID part at the end of the URN, designed exclusively for wearables. By selectively incorporating this token ID in the URN for wearables, we enable precise identification of each minted item, ensuring that no two minted wearables share the same URN within the Decentraland ecosystem.
+
+The purpose of this update is twofold:
+
+1. **Unambiguous Wearable Identification**: The inclusion of the token ID in the content-addressable URN guarantees unambiguous identification of each minted wearable NFT item within Decentraland. With the token ID, users, content creators, and developers can confidently validate ownership of specific minted wearable assets, accurately tracking their provenance and uniqueness.
+
+2. **Enhanced Ownership Validation**: The extended URN format significantly enhances the ownership validation process for minted wearables. By leveraging the unique token ID, the Catalyst's content validation system can effectively verify and confirm ownership of individual minted wearable items, streamlining interactions within the Decentraland metaverse.
+
+Let's consider an example where a user mints two identical blue shirts from a specific V2 collection. Currently, the URNs for each NFT appear as follows:
+* **First blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a
+* **Second blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a
+
+Once this ADR takes effect, the URNs will be modified as follows:
+* **First blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a:1
+* **Second blue shirt**: urn:decentraland:mumbai:collections-v2:0x02101c138653a0af06a45b729d9c5d6ba27b8f4a:2
+
+In these updated URNs, the last part of each URN (:1 and :2) represents the token ID, which becomes essential in identifying the specific blue shirt managed by the user.
+
+# Specification
+
+The extended content-addressable URN format for Decentraland wearables will be applied over collections v1 and v2 specifically. This extension was already proposed in the [ADR-109](/adr/ADR-109) on the second alternative:
+
+```
+decentraland:{protocol}:collections-v1:{contract(0x[a-fA-F0-9]+)}:{itemId}:{tokenId}
+```
+
+* **tokenID**: The unique identifier of the minted wearable NFT item, distinguishing it from other items of the same wearable content type and content identifier.
+
+The token ID part is optional and will only be used when addressing specific minted wearable NFT items. For other content types, such as scenes or other types of NFTs, the token ID part will not be added at the time of this writing.
```

Comment:
What about emotes? Should NFT emotes follow the same approach? In any case, scenes don't use urns for pointers. I think currently urns are only for wearables, emotes and stores.

Reply:
Added emotes to the ADR as discussed.